Sources/SwiftDocC/Utility/MarkupExtensions/BlockDirectiveExtensions.swift

```diff
@@ -40,6 +40,7 @@ extension BlockDirective {
         Stack.directiveName,
         Step.directiveName,
         Steps.directiveName,
+        SupportedLanguage.directiveName,
```

Comment:
Where is this list used? I had no idea this existed and haven't added my recent new directives to it 😅

Reply:
This is used for certain diagnostics IIRC, so that unrecognised directives are dropped.